### PR TITLE
Change menu option

### LINKED ISSUE #25: [Student A] Zmiana wymagania związanego z uzyskiwaniem pliku

### PR DESCRIPTION
**Pull request by KarolinaAntonowicz**
fix #25

### CODE REVIEW

File: analizator.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import requests
 import os
-menu = ['1. Download the file from the internet', '2. Count the number of letters in the downloaded file',
+menu = ['1. Wybierz plik wejsciowy', '2. Count the number of letters in the downloaded file',
```
Comment (naumowicz) [inline analizator.py]: Please change to english

File: analizator.py [hunk 2]
```diff
@@ -10,27 +10,40 @@
...
-        f.write(r.content)
-
-    # open file
-    file = open(filename, 'r')
-    text = list(file)
-    # text = file.read()
-    file.close()
-    return text
-
+    print("czy pobrać plik z internetu")
```
Comment (naumowicz) [inline analizator.py]: Please change to english

File: analizator.py [hunk 3]
```diff
@@ -10,27 +10,40 @@
...
-    text = list(file)
-    # text = file.read()
-    file.close()
-    return text
-
+    print("czy pobrać plik z internetu")  
+    action = (input())
+    if action == "Y":
+        global status_code
+        print("podaj adres źródłowy pliku")
```
Comment (naumowicz) [inline analizator.py]: Please change to english

File: analizator.py [hunk 4]
```diff
@@ -10,27 +10,40 @@
...
+        with open(filename, 'wb') as f:
+            f.write(r.content)
+        # open file
+        file = open(filename, 'r')
+        text = list(file)
+        # text = file.read()
+        file.close()
+        return text
+    if action == "N":
+         print("podaj nawę pliku txt")
```
Comment (naumowicz) [inline analizator.py]: Please change to english

**Comment by szymonsso** [comment]
plz fix this

File: analizator.py [hunk 5]
```diff
@@ -10,27 +10,40 @@
...
+         filename = input()
+         while (not os.path.exists(filename)) or (not os.path.exists(filename)):
```
Comment (naumowicz) [inline analizator.py]: unecessary loop

**Review by naumowicz — CHANGES_REQUESTED** [review]

**Review by naumowicz — APPROVED** [review]

**Review by szymonsso — APPROVED** [review]
good code mai friend

**Pull request merged**